### PR TITLE
MOTECH-2731: repair show error modal

### PR DESCRIPTION
**Pull request by palbecki**

### CODE REVIEW

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by jredlarski** [comment]
test this please

**Comment by kmadej** [comment]
@mkruszynski can you verify this PR ?

File: src/admin/bundles/bundle.routes.js [hunk 1]
```diff
@@ -40,7 +40,7 @@
         .state('bundles.bundleSettings', {
             url: 'bundleSettings/:bundleId',
             ncyBreadcrumb: {
-                label: 'admin.manageModuleSettings'
+                label: 'admin.settings'
```
Comment (mkruszynski) [inline src/admin/bundles/bundle.routes.js]: This breadcrumb fits better to admin settings page. Exists some reasons to use this?

File: src/admin/bundles/bundles.scss [hunk 2]
```diff
@@ -24,4 +24,8 @@
 
 .button-separator{
     margin: 0.5em;
+}
+
+legend{
```
Comment (mkruszynski) [inline src/admin/bundles/bundles.scss]: Don't style whole elements. Use prefixes to do this only for this module.

File: src/admin/bundles/bundle.routes.js (hunk 1 above)
Comment (palbecki) [inline src/admin/bundles/bundle.routes.js]: motech/motech#664

**Comment by jredlarski** [comment]
test this please

File: src/admin/bundles/bundles.scss [hunk 3]
```diff
@@ -24,4 +24,8 @@
 
 .button-separator{
     margin: 0.5em;
+}
+
+.legend-padding{
+    padding-bottom: 2px;
```
Comment (jredlarski) [inline src/admin/bundles/bundles.scss]: @palbecki Could you use `em` instead of magic numbers in `px`?

File: bower.json [hunk 4]
```diff
@@ -42,7 +42,8 @@
     "jquery-ui": "^1.11.4",
     "angular-cookies": "^1.5.7",
     "messageformat": "^0.3.1",
-    "jasny-bootstrap": "git://github.com/jasny/bootstrap"
+    "jasny-bootstrap": "git://github.com/jasny/bootstrap",
+    "jquery-i18n-properties": "git://github.com/jquery-i18n-properties/jquery-i18n-properties"
```
Comment (jredlarski) [inline bower.json]: @palbecki Why are you adding this to the bower.json file?

Comment (palbecki) [inline bower.json]: i18nService in bundles.factory.js use it

**Comment by jredlarski** [comment]
test this please

**Pull request merged**